app/scss/style.scss

```diff
@@ -115,16 +151,27 @@
       }
     }
   }
+  .specialties-list {
+    &:after {
+      content: "";
+      flex: auto;
```

Comment:
This + the `calc`s on the width at each breakpoint is the magic that makes the orphaned `li`s on the last row fall in line with the columns rather than flexing to fill the full space on the row. I think this is a pretty clean way of doing it (https://stackoverflow.com/questions/18744164/flex-box-align-last-row-to-grid), but maybe you know a better way?

Reply:
A good visual example of the difference here is in the before/after on the extra-large width, with the two on the final row staying on the left columns in the "After" version.